### PR TITLE
Add other users can checkout others sets

### PR DESCRIPTION
**Pull request by Branchverse**

### CODE REVIEW

File: src/set/set.service.ts [hunk 1]
```diff
@@ -86,29 +86,35 @@ export class SetService {
     }
 
     async getSetsFromUser(
```
Comment (LarsFlieger) [inline src/set/set.service.ts:88]: With this solution the user can not see his private stuff when he is visiting his profile url when logged in. (I mean it's not necessary yet)

**Comment by LarsFlieger** [comment]
Let's chat xD

**Review by LarsFlieger — CHANGES_REQUESTED** [review]

File: src/shared/dto/mongoId.dto.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { IsMongoId } from "class-validator";
-import { ObjectId } from "mongoose";
+import { IsMongoId, IsOptional } from 'class-validator';
+import { ObjectId } from 'mongoose';
 
 // Shared Dto to validate MongoId (We could create an own validation pipe for this)
 export class MongoIdDto {
-    
+    @IsOptional()
```
[the remark this answers is not in the document]
Comment (Branchverse) [reply src/shared/dto/mongoId.dto.ts]: Yes, Yes it is

File: src/set/set.service.ts (hunk 1 above)
Comment (Branchverse) [reply src/set/set.service.ts]: Now he can hehe

**Comment by LarsFlieger** [comment]
Looks Nice!

**Pull request merged**